src/super_gradients/training/transforms/transforms.py

```diff
@@ -567,7 +582,7 @@ def __call__(self, sample: dict):
             jit_factor = random.uniform(*self.mixup_scale)
 
             if len(img.shape) == 3:
-                cp_img = np.ones((self.input_dim[0], self.input_dim[1], 3), dtype=np.uint8) * 114
+                cp_img = np.ones((self.input_dim[0], self.input_dim[1], img.shape[2]), dtype=np.uint8) * 114
```

Comment:
if you added border_value, why is 114 hard codded here?

Reply:
what if if len(img.shape) == 2? (grayscale) I see you are checking that in line 607 but not here

Reply:
fixed- raised exception when channels < 3 in latest.

Reply:
Also added border value to all transforms where 114 was hard coded.